### ISSUE TITLE
cloning picture into sandbox using git bash help??

### ISSUE THREAD

**Issue by Jamielynn92** [ebeshero/DHClass-Hub#328]
What are my steps to doing it? I seem to get my commands confused? @ebeshero

**Comment by gabikeane** [comment]
Hi Jamie,

So if you want to add files to a specific repo in the shared DHClass-Hub, you should save the files in the Sandbox folder.  You can do that using File Explorer or Finder, the same way you would save to any other folder.  Then, open up git bash and navigate into the DHClass-Hub repo.  Remember the three basic commands for file navigation:

`cd`: change directory
`ls`: list files
`pwd`: print working directory (the current folder)

From there, you can use `git status` to see where you are.  Refer to the [Explain Git Shell](http://dh.newtfire.org/explainGitShell.html) for a walk through on how to commit your changes.  Remember to use `git status` each step of the way, so you can get a sense of what each command is doing.

**Comment by Jamielynn92** [comment]
![screenshot 6](https://user-images.githubusercontent.com/31480905/29998889-45d1b98e-9004-11e7-9aaf-d21dfc13d42e.png)
a classmate helped me with the steps to push the picture but I got denied. @ebeshero  @ajnewton1  @jonhoranic

**Comment by ebeshero** [comment]
@Jamielynn92 Uh oh--This was totally my fault and nothing you were doing wrong! It looks like I skipped adding your username to the DHClass-Hub last week, so I just did now. Check your email to accept my invitation, and then try this again! Sorry for the confusion!

**Comment by ebeshero** [comment]
And thanks for posting the screen shot! It helped me to figure out what was wrong!

**Comment by Jamielynn92** [comment]
how do I add you on there? it just takes me to my profile..

________________________________

**Comment by Jamielynn92** [comment]
Are we also suppose to close our repos as well?

**Comment by ebeshero** [comment]
@Jamielynn92 Sure, you'll want to clone any repo you create, so you make yourself a local workspace for it. You can practice with GitHub there too, maybe storing your homework files from class, or whatever you like. Later on, when we start work on HTML for websites, we'll ask you to use your personal GitHub repo to store your web files.

**Comment by Jamielynn92** [comment]
Will that be the same method as pushing the pictures or like my tinker bell repo already on git?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by ebeshero** [comment]
Your tinker bell repo is the one you made yourself on the web, so you can do a `git clone` to set it in your GitHub directory on your local computer, right next door to your clone of our DHClass-Hub. (Be careful that you clone it so it is inside the GitHub directory, and not down inside the DHClass-Hub, which will confuse the two repos.) 

Believe it or not, I have *thirty* repos cloned in my local GitHub directory! Lots of those are my copies of students' and project repos over a couple of years! :-D

**Comment by Jamielynn92** [comment]
![screenshot 9](https://user-images.githubusercontent.com/31480905/29999821-5dd49658-9024-11e7-9492-b2e4032024f0.png)
this is where it ended up for my first try but I'm not sure if its right.

**Comment by Jamielynn92** [comment]
Do I only need to write git clone in my terminal?

**Comment by ebeshero** [comment]
Yes--that's the command you type in your terminal followed by the cloud address of the repo you want to store on your local computer.

**Comment by Jamielynn92** [comment]
![screenshot 11](https://user-images.githubusercontent.com/31480905/29999953-9e51695a-9028-11e7-8618-f8754529e1e5.png)
what would be the proper way to type it out? 👍

**Comment by ebeshero** [comment]
You need the clone address! Where can you find it? (In your web browser, go find the web home of your repo and get its web address. There is a big green button on that page that is marked "Clone or Download". Click it to get the clone address, but it is really just the same as the website address for your repo.) I would paste this into your terminal. You may have trouble pasting in the way you are used to--so try right-clicking in the terminal window to bring up a menu with an option to paste. 

I am noticing that we could probably use a little detail on finding the clone address in our GitHub tutorial, but I do hope you are reading it! See http://dh.newtfire.org/explainGitShell.html#cloning

**Comment by kes213** [comment]
@Jamielynn92 I'm glad you created this issue because I was having problems committing a photo to the Sandbox myself! When I went back to the Explain Git Shell link and scrolled down to the "Basic Git Workflow" section, I realized I had skipped the command `git add` entirely. Maybe that's what you're doing too? That section was really helpful and I finally got my picture of Dwayne The Rock Johnson to push into the DH Class Hub!
